spec/builder/input_spec.rb

```diff
@@ -377,32 +377,32 @@ def length_should_be_required(options)
         end
 
         it 'should default to a string for methods on objects that don\'t respond to "column_for_attribute"' do
-          @new_post.stub!(:method_without_a_database_column)
-          @new_post.stub!(:column_for_attribute).and_return(nil)
+          @new_post.stub(:method_without_a_database_column)
+          @new_post.stub(:column_for_attribute).and_return(nil)
```

Comment:
+1 for moving from `stub!` to `stub` everywhere; according to the rspec author, `stub!` was an early mistake in the API.